lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -51,7 +51,7 @@ def self.auto_generate_changelog(repo_name, github_token, rate_limit_sleep, incl
 
         commits = Helper::GitHubHelper.get_commits_since_old_version(github_token, old_version, repo_name)
 
-        changelog_sections = { breaking_changes: [], new_features: [], paywalls: [], fixes: [], performance: [], dependency_updates: [], other: [] }
+        changelog_sections = { breaking_changes: [], new_features: [], paywalls: [], fixes: [], performance: [], dependency_updates: [], other: [], work_in_progress: {} }
```

Comment:
This might be just my opinion, but I don't think we should have a "WIP" section, since it doesn't provide much value to customer looking at the changelog and for us, we could just have them in an "Other" or "internal" section.

I think having custom sections (like Paywalls/Customer center) makes more sense for public facing sections of the changelog, which "WIP" would not be.

This might just be my opinion though. Happy to chat more about this over Slack!

Reply:
I agree these WIP doesn't hold value for most customers most of the time. Maybe we could put these inside [a toggle](https://gist.github.com/scmx/eca72d44afee0113ceb0349dd54a84a2) for us or others who may find them useful?

Reply:
those are valid points. I was also thinking of adding `feat:` as a prefix for arbitary features. For example `feat: Customer Center`, or `feat: Paywall Components`. Then things would be nested using the same sections we normally use (new features, fixes, other...). What do you think?

Reply:
Yeah, I think this makes total sense for `feat:` or `fix:`, but not for `wip:` IMO... I don't think we need a "pretty" changelog for ourselves, and customers won't care about anything in `wip:`. 

I would say that the changelog should mostly consist of 
- Breaking
- Feats (with any subsections per product)
- Bugfixes (with any subsections per product)
- Other (basically everything that the developer doesn't care about and/or is not user facing)

Again, happy to chat more about this though!